### PR TITLE
NMS-9900 Value of ${nodeLabel} for PSM services apparently not eagerly updated

### PR DESCRIPTION
**Pull request by patrick-schweizer**
* JIRA: https://issues.opennms.org/browse/NMS-9900

Our [continuous integration system](http://bamboo.internal.opennms.com:8085) will test and verify your changes.

### CODE REVIEW

File: opennms-model/src/main/java/org/opennms/netmgt/model/events/NodeLabelChangedEventBuilder.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import org.opennms.netmgt.events.api.EventConstants;
+import org.opennms.netmgt.xml.event.Event;
+import org.opennms.netmgt.xml.event.Parm;
+
+import com.google.common.base.Strings;
+
+public class NodeLabelChangedEventBuilder extends EventBuilder {
+
+    public NodeLabelChangedEventBuilder(final String source) {
+        super(EventConstants.NODE_LABEL_CHANGED_EVENT_UEI, source, new Date());
```
Comment (j-white) [inline opennms-model/src/main/java/org/opennms/netmgt/model/events/NodeLabelChangedEventBuilder.java]: Does this not trigger the `checkForIllegalUei()` check defined in the `EventBuilder` class?

Comment (patrick-schweizer) [reply opennms-model/src/main/java/org/opennms/netmgt/model/events/NodeLabelChangedEventBuilder.java]: you are right. I fixed it by overriding the method in the subclass

**Comment by patrick-schweizer** [comment]
@j-white , it looks to me that the bug exists since 2014, the parameters in the creation of the event were changed (https://github.com/OpenNMS/opennms/commit/444c1c05b7ed9d99718ac21be08c5544b6e3898b) but it looks like the processing of the event was still expecting the old parameter: PollerEventProcessor, line 382

**Comment by mvrueden** [comment]
Looks good to me as well.
This should at least go to `foundation-2017`, but may also be merged to `foundation-2016` or `foundation`. @j-white what do you think?

**Comment by j-white** [comment]
I agree. I'd say we re-target this for `foundation-2017`.

If the issue is raised on an older release we can consider back-porting it further then.

**Comment by patrick-schweizer** [comment]
ok will do

**Comment by patrick-schweizer** [comment]
Replaced by: #1984
=> closing this PR

**Pull request closed without merging**